udata_front/theme/gouvfr/assets/js/components/AdminAddMember/AdminAddMemberModal.vue

```diff
@@ -0,0 +1,144 @@
+<template>
+  <Teleport to="body">
+    <dialog
+      :aria-labelledby="modalTitleId"
+      role="dialog"
+      :id="id"
+      class="fr-modal"
+    >
+      <div class="fr-container fr-container--fluid fr-container-md">
+        <div class="fr-grid-row fr-grid-row--center">
+          <div class="fr-col-12 fr-col-md-8">
+            <div class="fr-modal__body">
+              <div class="fr-modal__header">
+                <button
+                  class="fr-btn--close fr-btn"
+                  :title="t('Close the modal dialog')"
+                  :aria-controls="id"
+                >
+                  {{ t("Close") }}
+                </button>
+              </div>
+              <form
+                @submit.prevent="submit"
+              >
+                <div class="fr-modal__content">
+                  <h1
+                    :id="modalTitleId"
+                    class="fr-modal__title fr-mb-2w"
+                  >
+                    {{ t("Add member to the organization") }}
+                  </h1>
+                  <MultiSelect
+                    :placeholder="$t('Search a user')"
+                    :searchPlaceholder="$t('Search a user...')"
+                    suggestUrl="/users/suggest/"
+                    :values="userId"
+                    @change="(value: string) => userId = value"
+                    :allOption="$t('Select a user')"
+                    :addAllOption="false"
+                    :onSuggest="formatUsers"
```

Comment:
When adding a member with suggest, I have too many calls made to `/avatars/../32 endpoint`.
I guess something is wrong with the multi-select, let me know if you reproduce.

Reply:
We have a max options shown arbitrarily set as 200 so we might want to reduce it to a saner value like 50

Reply:
Trying now with 30

Reply:
Changed to 20